invenio_assets/assets/build/webpack.config.js

```diff
@@ -252,6 +253,7 @@ var webpackConfig = {
   watchOptions: {
     followSymlinks: true,
   },
+  cache: process.env.NODE_ENV === "production",
```

Comment:
Major: Based on the docs this is by default false in `production`, I guess we should keep the same behaviour, unless I am missing something.
```suggestion
  cache: false,
```
Here the [docs](https://webpack.js.org/configuration/cache/).

Reply:
Understood @jrcastro2 , but the caching seems to default to true after first compile despite what the docs say. If setting it explicitly to false is preferred, I'm on board with that change.
On it.